docs/content/posts/2023-03-09-release-1.17.0.md

```diff
@@ -0,0 +1,487 @@
+was because a source operator with multiple splits might need to buffer a
+considerable amount of data if one split emitted data faster than another. To
+address this limitation, Flink 1.17 introduced [FLIP-217](https://cwiki.apache.org/confluence/display/FLINK/FLIP-217%3A+Support+watermark+alignment+of+source+splits),
+which enhances watermark alignment to align data emission across splits within
+a source operator while considering watermark boundaries. This enhancement
+ensures more coordinated watermark progress in the source, preventing downstream
+operators from buffering excessive data and improving the execution efficiency of
+streaming jobs. 
+
+
+## Streaming FileSink Expansion
+
+Following the addition of ABFS support, the
+[FileSink](https://nightlies.apache.org/flink/flink-docs-release-1.17/docs/connectors/datastream/filesystem/#file-sink)
+is now able to function in streaming mode with a total of five different
+filesystems: HDFS, S3, OSS, ABFS, and Local. This expansion effectively covers
+the majority of main filesystems, providing a comprehensive range of options and
+increased versatility for users.
+
+
+## RocksDBStateBackend Upgrade
+
+This release has updated the version of
+[FRocksDB](https://github.com/ververica/frocksdb) to 6.20.3-ververica-2.0 which
+brings improvements for RocksDBStateBackend:
+
+
+1. Support build FRocksDB Java on Apple Silicon chipsets, such as Mac M1 and M2.
+2. Improve the performance of compaction filter by avoiding expensive ToString()
+3. Upgrade ZLIB version of FRocksDB to avoid memory corruption
+4. Add periodic_compaction_seconds option to RocksJava
+
+Please see [FLINK-30836](https://issues.apache.org/jira/browse/FLINK-30836) for
+more details.
+
+This release also widens the scope of sharing memory between slots to
+TaskManager, which can help to increase the memory efficiency if the memory
+usage of slots in a TaskManager is uneven. Furthermore, it can reduce the
+overall memory consumption at the expense of resource isolation after tuning.
+Read more about
+[state.backend.rocksdb.memory.fixed-per-tm](https://nightlies.apache.org/flink/flink-docs-release-1.17/docs/deployment/config/#state-backend-rocksdb-memory-fixed-per-tm)
+configuration. 
+
+
+## Calcite Upgrade
+
+Flink 1.17 is upgraded to [Calcite](https://calcite.apache.org/) version 1.29.0
+to improve the performance and efficiency of the Flink SQL system. Flink 1.16
+uses Calcite 1.26.0 which has severe issues with RexNode simplification caused
+by the SEARCH operator. This leads to wrong data from query optimization as
+reported in [CALCITE-4325](https://issues.apache.org/jira/browse/CALCITE-4325) and [
+CALCITE-4352](https://issues.apache.org/jira/browse/CALCITE-4352). By upgrading
+the version of Calcite, Flink can take advantage of its improved
+performance and new features in Flink SQL processing. This resolves multiple
+bugs and leads to faster query processing times. The downside of this upgrade is
```

Comment:
As suggested in the discussion of apache/flink#22146, what about removing the statement `The downside of this upgrade is that the execution plan changes for certain SQL queries. Savepoints are not backwards compatible in these cases` ? This incompatibility should have already been highlighted in the [doc](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/concepts/overview/#stateful-upgrades-and-evolution) and the coming release note of 1.17.

Reply:
Make sense to me